jobs/announce_events_happening_today.js

```diff
@@ -25,8 +26,8 @@ module.exports = (channel, iconUrl, bot) => {
           attachments.push({
             fields: [
               { title: 'Location', value: event.location },
-              { title: 'Start At', value: event.start.dateTime, short: true },
-              { title: 'Ends At', value: event.end.dateTime, short: true },
+              { title: 'Start At', value: format(event.start.dateTime, [format='HH:mm']), short: true },
```

Comment:
Looks like you are only formatting to Hour and Minute, and without the actual Day, Month, Year.